dist/index.js

```diff
@@ -3430,37 +3420,53 @@ function run() {
             // ament_cmake should handle this automatically, but we are seeing cases
             // where this does not happen. See issue #26 for relevant CI logs.
             core.addPath(path.join(rosWorkspaceDir, "install", "bin"));
+            // Source any installed ROS distributions if they are present
+            let colconCommandPrefix = "";
+            if (isLinux) {
+                if (targetRos1Distro) {
+                    const ros1SetupPath = `/opt/ros/${targetRos1Distro}/setup.sh`;
+                    if (fs_1.default.existsSync(ros1SetupPath)) {
+                        colconCommandPrefix += `source ${ros1SetupPath} && `;
```

Comment:
It seems weird to me that `colconCommandPrefix` is responsible for the `&&`: shouldn't this be done in `execBashCommand`?

Reply:
not necessarily, you could potentially make the prefix `ENV_VAR=something ` and that would be a valid use

Reply:
Since colconCommandPrefix is optional, it might be best to leave it in here.